clang/test/CodeGenCXX/debug-info-incomplete-types.cpp

```diff
@@ -0,0 +1,12 @@
+// RUN: %clang_cc1 -debug-info-kind=limited -gomit-unreferenced-members %s -emit-llvm -o - | FileCheck %s
```

Comment:
Test needs renaming for the different flag name?

Reply:
Done